config/common/errors.go

```diff
@@ -37,6 +37,8 @@ var (
 	ErrNodeExists             = errors.New("matching filesystem node has existing contents or different type")
 	ErrNoFilesDir             = errors.New("local file paths are relative to a files directory that must be specified with -d/--files-dir")
 	ErrTreeNotDirectory       = errors.New("root of tree must be a directory")
+	ErrRootTooSmall           = errors.New("root should have 8GiB of space available")
+	ErrRootNotLastPartition   = errors.New("root should be last partition number to allow for growth")
```

Comment:
The message is misleading, since we allow and encourage adding partitions after the root.